Modix.Bot/Modules/FunModule.cs

```diff
@@ -31,22 +33,83 @@ public FunModule(IHttpClientFactory httpClientFactory)
                 var req = await client.GetStreamAsync(emojiUrl);
 
                 await Context.Channel.SendFileAsync(req, Path.GetFileName(emojiUrl), Context.User.Mention);
-                
+
                 try
                 {
                     await Context.Message.DeleteAsync();
                 }
-                catch (HttpRequestException)
+                catch (HttpRequestException ex)
                 {
-                    Log.Information("Couldn't delete message after jumbofying.");
+                    Log.Warning(ex, "Couldn't delete message after jumbofying.");
                 }
             }
-            catch (HttpRequestException)
+            catch (HttpRequestException ex)
             {
+
+                Log.Warning(ex, "Failed jumbofying emoji");
                 await ReplyAsync($"Sorry {Context.User.Mention}, I don't recognize that emoji.");
             }
         }
 
+        [Command("avatar"), Alias("av", "ava", "pfp"), Summary("Gets an avatar for a user")]
+        public async Task GetAvatarAsync(
+            [Summary("User that has the avatar")]
+                IUser user,
+                [Summary("Size for the avatar, defaults to 128")]
+                ushort size = 128)
+        {
+
+            try
+            {
+                const ushort MiniumSize = 128;
```

Comment:
Worth making a config setting?

Reply:
Also, before this gets merged, it'd be worth it to change that to MinimumSize